python/mhs/generic_scenarios.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (c) 2019, GEM Foundation.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.
+# If not, see <https://www.gnu.org/licenses/agpl.html>.
+#
+import json
+import sys
+
+from mhs import Footprint, FootprintSet, Event, EventSet
+
+from import_scenarios import import_event_set
+
+
+VERBOSE = True
+
+
+def verbose_message(msg):
+    """
+    Display message if we are in verbose mode
+    """
+    if VERBOSE:
+        sys.stderr.write(msg)
+
+
+class JSONEventSet():
+    """
+    Read meta-data for EventSets from JSON
+    """
+    def __init__():
+        pass
```

Comment:
This is very strange. How wonder how the class can be instantiated since the self if missing. But even if it was there, what's the point of this `__init__`?

Reply:
Ah, I see, this is a class with only classmethods that is never instantiated. It is a bit Java-like, in Python normally you would just use functions but it is not a problem.